<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TournamentTab</class>
 <widget class="QWidget" name="TournamentTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="container" stretch="2,0,2">
     <item>
      <layout class="QVBoxLayout" name="left" stretch="1,1,1,1,0">
       <item>
        <widget class="QLabel" name="tournamentStatus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Tournament Name and Type</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="tournamentSettings">
           <property name="text">
            <string>Settings</string>
           </property>
           <property name="icon">
            <iconset theme="preferences-system">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pairRound">
           <property name="text">
            <string>Pair Round</string>
           </property>
           <property name="icon">
            <iconset theme="go-next">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Matches</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="roundTable" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="middle" stretch="1">
       <item>
        <widget class="QTabWidget" name="infoTabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>100</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="right" stretch="1,1,1">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Players</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="playerTable" native="true"/>
       </item>
       <item>
        <widget class="QPushButton" name="addPlayer">
         <property name="text">
          <string>Add Player</string>
         </property>
         <property name="icon">
          <iconset theme="folder-new">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statusBar" stretch="0,0,0,0,0,0">
     <item>
      <widget class="QCheckBox" name="startedIndicator">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Not Started</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusIndicator">
       <property name="text">
        <string>Tournament Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="roundTimerLabel">
       <property name="text">
        <string>Time Left in Round</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
       <property name="format">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
